### PR TITLE
updates to view subscription - VPC peering

### PR DESCRIPTION
**Pull request by Aviad-Abutbul**

### CODE REVIEW

File: content/rv/administration/setup_and_editing/view-edit-subscription.md [hunk 1]
```diff
@@ -41,23 +41,20 @@ databases.
...
 After you create a Redis Enterprise VPC (RV) subscription, you can configure VPC
-peering and a CIDR whitelist to allow more direct access to your VPC. VPC
-peering and CIDR whitelists are capabilities provided by [Amazon Virtual Private Cloud]
-(https://docs.aws.amazon.com/vpc/latest/userguide/) (Amazon VPC), but you can
-configure them here in RV.
+peering and a CIDR whitelist to allow more secure access to your VPC.
 
 - [VPC peering](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html) -
-    The route tables of each VPC have access to the entire CIDR block of the peered VPC.
+    Connect two VPCs together, allowing network access between them.
```
Comment (bmansheim) [inline content/rv/administration/setup_and_editing/view-edit-subscription.md]: How about: Allow unresticted network access between the VPCs.

File: content/rv/administration/setup_and_editing/view-edit-subscription.md [hunk 2]
```diff
@@ -41,23 +41,20 @@ databases.
...
 - [CIDR whitelist](https://docs.aws.amazon.com/vpc/latest/peering/peering-configurations-partial-access.html) -
-    The route table of your VPC has access to the specified CIDR blocks and security groups.
+    narrow down the access to your entire subscription by specifing the relevant CIDR blocks and security groups.
```
Comment (bmansheim) [inline content/rv/administration/setup_and_editing/view-edit-subscription.md]: Limit network access between the VPCs to the specified CIDR blocks and security groups.

File: content/rv/administration/setup_and_editing/view-edit-subscription.md [hunk 3]
```diff
@@ -41,23 +41,20 @@ databases.
...
 
 ### VPC Peering
 
 A VPC peering connection is a networking connection between two VPCs that enables you
 to route traffic between them using private IP addresses. Instances in either VPC can
 communicate with each other as if they are within the same network. You can connect your
 VPC in the RV subscription to the VPC of your application. Then your application can
-connect to the RV database using VPC Peering to optimize the performance of your application.
+connect securly to your RV database using VPC Peering to optimize the performance of your application.
```
Comment (bmansheim) [inline content/rv/administration/setup_and_editing/view-edit-subscription.md]: securely

File: content/rv/administration/setup_and_editing/view-edit-subscription.md (hunk 2 above)
Comment (allegrass) [reply content/rv/administration/setup_and_editing/view-edit-subscription.md]: Sounds good

File: content/rv/administration/setup_and_editing/view-edit-subscription.md (hunk 1 above)
Comment (allegrass) [reply content/rv/administration/setup_and_editing/view-edit-subscription.md]: Between two VPCs

**Pull request merged**